src/seedu/addressbook/data/tag/UniqueTagList.java

```diff
@@ -94,18 +94,6 @@ public boolean contains(Tag toCheck) {
     }
 
     /**
-     * Adds a Tag to the list.
-     *
-     * @throws DuplicateTagException if the Tag to add is a duplicate of an existing Tag in the list.
-     */
-    public void add(Tag toAdd) throws DuplicateTagException {
```

Comment:
> this should probably be in a separate commit

Agree.

> whether it is a good reason to remove UniqueTagList#add() just because AddressBook#addTag() no longer exist

Since UniqueTagList#add() is never used as well. I would suggest we remove it.